app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -364,6 +358,8 @@ class MainActivity : AppCompatActivity() {
             VisionSafetyManager.create(VisionManager, visionSafetyListener)
 
             visionManagerWasInit = true
+
+            debug_view.startSessionTime()
```

Comment:
Guess session time isn't needed for VisionManager, sorry for confusion. It's relevant only for Replay mode, when there is a session and it's timeline.